app/models/ontology/scopes.rb

```diff
@@ -2,15 +2,72 @@ module Ontology::Scopes
   extend ActiveSupport::Concern
 
   included do
+    equal_scope *%w(
```

Comment:
Lint/AmbiguousOperator: Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the `*` if it should be a multiplication.